docs/examples/code/working_with_signed_urls.md

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
Could probably use a top-level header?

```
# Example with Signed URLs
```

Reply:
I'm not sure to understand @kylebarron

Reply:
This document (in isolation) doesn't have a title. Maybe mkdocs adds a title if you don't provide an initial `h1` header in the markdown?

Reply:
yeah, mkdocs adds a title. I think it's fine as it is